tutorial/source/intro_part_ii.ipynb

```diff
@@ -247,8 +247,8 @@
     "def scale_obs(guess):\n",
```

Comment:
Should just delete this function now that it's identical to the example just above it

Reply:
Good catch. Will remove this.